docs/datastorage/nbt.md

```diff
@@ -0,0 +1,98 @@
+# Named Binary Tag (NBT)
+
+NBT is a format introduced in the earliest days of Minecraft, written by Notch himself. It is widely used throughout the Minecraft codebase for data storage.
+
+## Specification
+
+The NBT spec is similar to the JSON spec, with a few differences:
+
+- Distinct types for bytes, shorts, longs and floats exist, suffixed by `b`, `s`, `l` and `f`, respectively, similar to how they would be represented in Java code.
+  - Doubles may also be suffixed with `d`, but this is not required, similar to Java code. The optional `i` suffix available in Java for integers is not permitted.
+  - All of these suffixes may also be capital letters. So for example, `64b` is the same as `64B`, and `0.5F` is the same as `0.5f`.
```

Comment:
Not case sensitive?

Reply:
If the Minecraft wiki is anything to go by, no.

Reply:
Sorry, I mean this as a way to word the sentence. Like, 'the suffixes are case insensitive.'